corehq/apps/export/tests/test_incremental.py

```diff
@@ -160,9 +161,14 @@ def test_incremental_success(self):
 
         self.assertEqual(self.incremental_export.checkpoints.count(), 3)
 
+    # TODO: Fix the below 2 tests. They are failing because the address they are using (http://somewhere)
+    # is unresolvable, and now raises an exception. Untangling and mocking the appropriate calls out
```

Comment:
That was one of my 3 failed attempts at fixing this. Using a URL that did resolve caused issues somewhere else, because the rest of the tests relied on something else being `http://somewhere`

Reply:
Just replaced `http://somewhere` in both locations. I still don't love that these tests are making real requests.